doc/sfsmaster.cfg.5.adoc

```diff
@@ -238,6 +238,13 @@ chunkserver disk usage to determine most fitting chunkserver. Heavy loaded
 chunkservers will be picked for operations less frequently. (default is 0,
 correct values are in range from 0 to 0.5)
 
+*ENABLE_PROMETHEUS*:: Whether to enable Prometheus support and metric
+collection. Note that this requires compiling with Prometheus support. Set to
+either 1 to enable, or 0 to disable (default is 0)
+
+*PROMETHEUS_HOST*:: Host address where Prometheus metric data can be collected,
+must be in the format of HOST:PORT (default 0.0.0.0:8080)
```

Comment:
Port 8080 is a highly used one. Please consider selecting a default one less prone to conflicts with other applications. Probably 94XX like other saunafs components.

Reply:
I went with 9499 for now.